api/storage/client_test.go

```diff
@@ -624,3 +624,40 @@ func (s *storageMockSuite) TestDestroyInvalidStorageId(c *gc.C) {
 	_, err := client.Destroy([]string{"foo/bar"})
 	c.Check(err, gc.ErrorMatches, `storage ID "foo/bar" not valid`)
 }
+
+func (s *storageMockSuite) TestAttach(c *gc.C) {
```

Comment:
Many of our API tests also have a test for "expected 5 results got 6" type thing